models.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package wptdashboard
+
+import (
+    "time"
+)
+
+type TestRun struct {
+    // Platform information
+    BrowserName     string          `json:"browser_name"`
+    BrowserVersion  string          `json:"browser_version"`
+    OSName          string          `json:"os_name"`
+    OSVersion       string          `json:"os_version"`
+
+    // The full SHA1 of the tested WPT revision
+    Revision        string          `json:"revision"`
+
+    // Results URL
+    ResultsURL      string          `json:"results_url"`
+
+    CreatedAt       time.Time       `json:"created_at"`
+}
```

Comment:
`TestRun` is a model that isn't (yet) described in the design doc, but I think it fits our use case more optimally. Every time a test run finishes, it'll create a new TestRun. Then, when someone loads a page we can query the latest TestRun by each platform.